package.json

```diff
@@ -92,19 +104,45 @@
         {
           "command": "memory-inspector.show-variable",
           "when": "false"
+        },
+        {
+          "command": "memory-inspector.store-file",
```

Comment:
We should disable the commands/buttons in the window if no debug session is active.
At the moment, I can click the buttons outside the debug session and get errors as expected.

Reply:
That is a very good point! 

We can see that the command is already disabled as it does not show up in the command palette anymore (F1 + 'memory' because we can no longer read or write. I moved the enablement condition directly to the command but it seems that VS code still allows us to simply execute the command via code and I don't see any other way to determine whether our command is enabled or not. The documentation even states it like that:

```
(Optional) Condition which must be true to enable the command in the UI (menu and keybindings). Does not prevent executing the command by other means, like the `executeCommand`-api.
```

I added a new commit that now shares the current session state with the memory inspector so we can properly determine the read/write capabilities and disable our buttons accordingly.